Features/HideProtectedPosts/functions.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Flynt\Features\HideProtectedPosts;
+
+add_action('pre_get_posts', function ($query) {
+    if (!$query->is_singular() && !is_admin()) {
+        $query->set('has_password', false);
```

Comment:
I got it from the `WP_Query` doc:
https://codex.wordpress.org/Class_Reference/WP_Query#Password_Parameters
I've tested it, it works perfectly fine.
Should I modify this?